src/main/resources/application.yml

```diff
@@ -88,18 +88,6 @@ data-grid:
 
   report-lock-time-to-live-seconds: 300
 
-# the thread that sends actions to handlers
```

Comment:
Is it not being used by scheduler?

Reply:
That was used by the old version of the scheduler (with redis and the @scheduled annotation).  The new one has its timing set in the helm chart